doc/classes/ProjectSettings.xml

```diff
@@ -2890,11 +2890,11 @@
 			If [code]true[/code], forces vertex shading for all rendering. This can increase performance a lot, but also reduces quality immensely. Can be used to optimize performance on low-end mobile devices.
 		</member>
 		<member name="rendering/textures/canvas_textures/default_texture_filter" type="int" setter="" getter="" default="1">
-			The default texture filtering mode to use on [CanvasItem]s.
+			The default texture filtering mode to use for [CanvasItem]s built-in texture. In shaders, this texture is accessed as [code]TEXTURE[/code].
```

Comment:
```suggestion
			The default texture filtering mode used to render the built-in texture for [CanvasItem] nodes. In shaders, this texture is accessed as [code]TEXTURE[/code].
```

Reply:
Notably `texture` is not a property of `CanvasItem` - it's a property of several nodes that inherit `CanvasItem`:
https://docs.godotengine.org/en/stable/classes/class_texturerect.html#class-texturerect-property-texture
https://docs.godotengine.org/en/stable/classes/class_sprite2d.html#class-sprite2d-property-texture
https://docs.godotengine.org/en/stable/classes/class_line2d.html#class-line2d-property-texture
If this note goes on `texture`, it would have to be repeated in several places.

I also think that the project settings are a reasonable place to put general information about Godot that don't fit anywhere else in the class reference. At least that's how they are used currently. This note seems *fine* to me.

Reply:
Also seems fine to me. :)